examples/sbt/single-module/src/main/scala/Example.scala

```diff
@@ -1,3 +1,18 @@
 object Example {
   def sum(a: Int, b: Int) = a + b
+
+  // An example method full of horrible style issues!
+  // This is only indended for demonstration purposes.
+  // Please don't do this at home, because every time
+  // you write code like this, a cute kitten dies!
```

Comment:
This comment has my dying 😸

Reply:
I've must have literally killed a dozen kittens by writing this code 😢